### ISSUE TITLE
Add "request_path" parameter for the Roundcube application

### ISSUE THREAD

**Issue by andy-walker321** [truenas/charts#2223]
This is a suggestion request for TrueNAS Roundcube application.
Roundcube parameter "request_path" is very useful in case when Roundcube runs behind reverse proxy.
In such case user can configure reverse proxy to send custom header (e.g. X-Forwarded-Path) containing the path under which Roundcube is exposed to the outside world. This could be used by Roundcube as "request_path" parameter. In current configuration "request_path" is null and hence custom header sent by reverse proxy is ignored.
It should be relatively easy to do by making "REQUESTED_PATH" available through application configuration interface in TrueNAS and consequently replacing $config['request_path'] = '/'; in config.docker.inc.php by whatever user configured in web interface.
For example, user configured "REQUESTED_PATH" = /rcube/. Which leads to $config['request_path'] = '/rcube/' in docker config file. Then user also use X-Forwarded-Path custom header in his reverse proxy.
Reference: from roundcubemail-1.6.6/config/defaults.inc.php 
  806 // Specifies the full path of the original HTTP request, either as a real path or
  807 // $_SERVER field name. This might be useful when Roundcube runs behind a reverse
  808 // proxy using a subpath. This is a path part of the URL, not the full URL!
  809 // The reverse proxy config can specify a custom header (e.g. X-Forwarded-Path) containing
  810 // the path under which Roundcube is exposed to the outside world (e.g. /rcube/).
  811 // This header value is then available in PHP with $_SERVER['HTTP_X_FORWARDED_PATH'].
  812 // By default the path comes from  'REDIRECT_SCRIPT_URL', 'SCRIPT_NAME' or 'REQUEST_URI',
  813 // whichever is set (in this order).
  814 $config['request_path'] = null;

**Comment by stavros-k** [comment]
Hello, you can add this as a variable in the `Additional Environment Variables` section

Other available variables are here:
https://github.com/roundcube/roundcubemail-docker?tab=readme-ov-file#configurationenvironment-variables

**Comment by andy-walker321** [comment]
Indeed. Sorry, I missed it. I was checking those environmental variables but because that one called slightly differently ROUNDCUBEMAIL_REQUEST_PATH I missed it. My bad.
With that env. var. set it works fine under reverse proxy.
